PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Pull request title
```

Comment:
Title is already outside the template, this would duplicate the title

Reply:
Pull request title removed.